errors.go

```diff
@@ -372,4 +372,8 @@ var (
 	ErrRequiredRawState = errors.New("RawState is required")
 
 	ErrStateVersionUploadNotSupported = errors.New("upload not supported by this version of Terraform Enterprise")
+
+	ErrPrivateVCSDisabled = errors.New("the organization does not have private VCS enabled")
+
+	ErrAgentPoolUnavailable = errors.New("the provided agent pool does not exist or you are not authorized to use it")
```

Comment:
Really minor issue but these appear to only be used in assertions in your test code and never returned as errors by go-tfe. You could avoid the public interface and just compare these in your assertions:

`assert.Contains(t, errCreate.Error(), "the organization does not have private VCS enabled")`